src/applications/hca/tests/e2e/utils/helpers.js

```diff
@@ -14,11 +14,14 @@ export const goToNextPage = pagePath => {
   }
 };
 
-export const startAsAuthUser = () => {
+export const startAsAuthUser = (opts = {}) => {
```

Comment:
Let's simplify this down to automatically destruct the opts object. We don't want to have `startAsAuthUser(false)` ran with zero context of what `false` means.